### PR TITLE
Pre v0.7.2 fixes

### PR DESCRIPTION
**Pull request by beevital**
Pre-release version:
- connector OAS-based resource disabled

### CODE REVIEW

File: fivetran/resource_connector_legacy.go [hunk 1]
```diff
@@ -0,0 +1,1229 @@
...
+	if len(auth) < 1 {
+		return &authMap
+	}
+	if auth[0] == nil {
+		return &authMap
+	}
+
+	// add custom auth fields here:
+
+	// a := auth[0].(map[string]interface{})
```
Comment (Fivetran-RomanKolesnikov) [inline fivetran/resource_connector_legacy.go:383]: Is it not needed?

File: fivetran/tests/mock/datasource_connector_test.go [hunk 2]
```diff
@@ -121,7 +121,7 @@ func TestDataSourceConnectorConfigMappingMock(t *testing.T) {
 			resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.port", "0"),
 			resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.tunnel_port", "0"),
 			resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.api_quota", "0"),
-			resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.network_code", "0"),
+			// resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.network_code", "0"),
```
Comment (Fivetran-RomanKolesnikov) [inline fivetran/tests/mock/datasource_connector_test.go:124]: should it be removed?

File: fivetran/tests/mock/datasource_connector_test.go [hunk 3]
```diff
@@ -329,7 +329,7 @@ func TestDataSourceConnectorConfigMappingMock(t *testing.T) {
 			resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.custom_tables.0.click_attribution_window", "click_attribution_window"),
 			resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.custom_tables.0.view_attribution_window", "view_attribution_window"),
 			resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.custom_tables.0.prebuilt_report_name", "prebuilt_report_name"),
-			resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.custom_tables.0.use_unified_attribution_setting", "false"),
+			//resource.TestCheckResourceAttr("data.fivetran_connector.test_connector", "config.0.custom_tables.0.use_unified_attribution_setting", "false"),
```
Comment (Fivetran-RomanKolesnikov) [inline fivetran/tests/mock/datasource_connector_test.go:332]: should it be removed?

File: fivetran/tests/mock/resource_connector_mapping_test.go [hunk 4]
```diff
@@ -0,0 +1,1195 @@
...
+			assertKeyExists(t, config, "packed_mode_tables")
+			assertArrayItems(t, config["packed_mode_tables"].([]interface{}), append(make([]interface{}, 0), "packed_mode_table"))
+
+			assertKeyExists(t, config, "properties")
+			assertArrayItems(t, config["properties"].([]interface{}), append(make([]interface{}, 0), "property"))
+
+			assertKeyExists(t, config, "primary_keys")
+			assertArrayItems(t, config["primary_keys"].([]interface{}), append(make([]interface{}, 0), "primary_key"))
+
+			// assertKeyExists(t, config, "app_ids")
```
Comment (Fivetran-RomanKolesnikov) [inline fivetran/tests/mock/resource_connector_mapping_test.go:1004]: Don't we need to check these fields?

**Review by Fivetran-RomanKolesnikov — APPROVED** [review]
Looks good!
I found some commented code, maybe it should be removed or to be uncommented

File: fivetran/tests/mock/datasource_connector_test.go (hunk 3 above)
Comment (beevital) [reply fivetran/tests/mock/datasource_connector_test.go:332]: no, it's for OAS-schema coverage

File: fivetran/tests/mock/resource_connector_mapping_test.go (hunk 4 above)
Comment (beevital) [reply fivetran/tests/mock/resource_connector_mapping_test.go:1004]: Same here

File: fivetran/resource_connector_legacy.go (hunk 1 above)
Comment (beevital) [reply fivetran/resource_connector_legacy.go:383]: it's just an example code, now we don't have custom auth fields.

**Pull request merged**